### ISSUE TITLE
Requirements are missing in readme

### ISSUE THREAD

**Issue by vguleria1108** [komoot/photon#523]
I am trying to deploy photon on elasticbean but my instance keeps stopping, so if you can add minimum requirements required than it will be helpful.

**Comment by kenseii** [comment]
Hi @vguleria1108 did you find the answer for this? i am trying to deploy photon as well

**Comment by lonvia** [comment]
The instance at photon.komoot.io currently runs at a machine with 64GB RAM and 6 CPUs. But that's on the upper end of what you need. 32GB RAM will do for less busy installations, maybe even 16GB are enough. I have no reliable numbers for minimum requirements which is why the ticket is still open.

This is all for running a planet-wide photon from the pre-computed data dumps we provide. If you want to create the database yourself, the [hardware requirements for Nominatim](https://nominatim.org/release-docs/latest/admin/Installation/#hardware) apply.

**Comment by akatie** [comment]
The deployment was successful.

**Comment by lonvia** [comment]
On what kind of hardware?

**Comment by otbutz** [comment]
@lonvia we're running Photon successfully with a Europe dataset using `-Xms2G -Xmx2G` on VMs with at least 4 cores and 12GB RAM. For the duration of the import we're using `-Xms5G -Xmx5G`.

**Comment by henrik242** [comment]
Are there any plans on raising the JDK requirement to the latest LTS (21)?

**Comment by lonvia** [comment]
No. Policy is to support the three latest LTS unless there is a requirement from the dependencies that cannot be worked around. So, we probably raise to 17 by end of year but certainly not to 21.

**Comment by henrik242** [comment]
I have usually been able to get newer jdks on older distros without issues, though.

**Comment by lonvia** [comment]
Overhauled the README. The recommendations remain vague but that's as godd as it gets because requirements depend a lot on the kind of load the service gets.
